Cargo.lock

```diff
@@ -1721,18 +1711,18 @@ dependencies = [
 
 [[package]]
 name = "semver"
-version = "1.0.17"
+version = "1.0.18"
 source = "registry+https://github.com/rust-lang/crates.io-index"
-checksum = "bebd363326d05ec3e2f532ab7660680f3b02130d780c299bca73469d521bc0ed"
+checksum = "b0293b4b29daaf487284529cc2f5675b8e57c61f70167ba415a463651fd6a918"
 dependencies = [
  "serde",
 ]
 
 [[package]]
 name = "serde"
-version = "1.0.164"
+version = "1.0.180"
```

Comment:
serde 1.0.172+ cannot be built with Nix (see https://github.com/serde-rs/serde/issues/2538). Please downgrade it to version 1.0.171:

```shell
cargo update --package serde --precise 1.0.171
```